{
	"document": {
		"aggregate_severity": {
			"namespace": "https://www.suse.com/support/security/rating/",
			"text": "moderate"
		},
		"category": "csaf_security_advisory",
		"csaf_version": "2.0",
		"distribution": {
			"text": "Copyright 2023 SUSE LLC. All rights reserved.",
			"tlp": {
				"label": "WHITE",
				"url": "https://www.first.org/tlp/"
			}
		},
		"lang": "en",
		"notes": [
			{
				"category": "summary",
				"text": "Security update for python-markdown2",
				"title": "Title of the patch"
			},
			{
				"category": "description",
				"text": "This update for python-markdown2 fixes the following issues:\n\n- CVE-2020-11888: Fixed unsanitized input for cross-site scripting (boo#1171379)\n",
				"title": "Description of the patch"
			},
			{
				"category": "details",
				"text": "openSUSE-2020-651",
				"title": "Patchnames"
			},
			{
				"category": "legal_disclaimer",
				"text": "CSAF 2.0 data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
				"title": "Terms of use"
			}
		],
		"publisher": {
			"category": "vendor",
			"contact_details": "https://www.suse.com/support/security/contact/",
			"name": "SUSE Product Security Team",
			"namespace": "https://www.suse.com/"
		},
		"references": [
			{
				"category": "self",
				"summary": "URL for openSUSE-SU-2020:0651-1",
				"url": "https://lists.opensuse.org/archives/list/security-announce@lists.opensuse.org/thread/B4CURXVTATQIUIFDWQXVQYUATP43MCQ2/"
			},
			{
				"category": "external",
				"summary": "SUSE ratings",
				"url": "https://www.suse.com/support/security/rating/"
			},
			{
				"category": "self",
				"summary": "URL of this CSAF notice",
				"url": "https://ftp.suse.com/pub/projects/security/csaf/opensuse-su-2020_0651-1.json"
			},
			{
				"category": "self",
				"summary": "E-Mail link for openSUSE-SU-2020:0651-1",
				"url": "https://lists.opensuse.org/archives/list/security-announce@lists.opensuse.org/thread/B4CURXVTATQIUIFDWQXVQYUATP43MCQ2/"
			},
			{
				"category": "self",
				"summary": "SUSE Bug 1171379",
				"url": "https://bugzilla.suse.com/1171379"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2020-11888 page",
				"url": "https://www.suse.com/security/cve/CVE-2020-11888/"
			}
		],
		"title": "Security update for python-markdown2",
		"tracking": {
			"current_release_date": "2020-05-11T12:19:43Z",
			"generator": {
				"date": "2020-05-11T12:19:43Z",
				"engine": {
					"name": "cve-database.git:bin/generate-csaf.pl",
					"version": "1"
				}
			},
			"id": "openSUSE-SU-2020:0651-1",
			"initial_release_date": "2020-05-11T12:19:43Z",
			"revision_history": [
				{
					"date": "2020-05-11T12:19:43Z",
					"number": "1",
					"summary": "Current version"
				}
			],
			"status": "final",
			"version": "1"
		}
	},
	"product_tree": {
		"branches": [
			{
				"branches": [
					{
						"branches": [
							{
								"category": "product_version",
								"name": "python2-markdown2-2.3.7-lp151.2.3.1.noarch",
								"product": {
									"name": "python2-markdown2-2.3.7-lp151.2.3.1.noarch",
									"product_id": "python2-markdown2-2.3.7-lp151.2.3.1.noarch"
								}
							},
							{
								"category": "product_version",
								"name": "python3-markdown2-2.3.7-lp151.2.3.1.noarch",
								"product": {
									"name": "python3-markdown2-2.3.7-lp151.2.3.1.noarch",
									"product_id": "python3-markdown2-2.3.7-lp151.2.3.1.noarch"
								}
							}
						],
						"category": "architecture",
						"name": "noarch"
					},
					{
						"branches": [
							{
								"category": "product_name",
								"name": "openSUSE Leap 15.1",
								"product": {
									"name": "openSUSE Leap 15.1",
									"product_id": "openSUSE Leap 15.1",
									"product_identification_helper": {
										"cpe": "cpe:/o:opensuse:leap:15.1"
									}
								}
							}
						],
						"category": "product_family",
						"name": "SUSE Linux Enterprise"
					}
				],
				"category": "vendor",
				"name": "SUSE"
			}
		],
		"relationships": [
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python2-markdown2-2.3.7-lp151.2.3.1.noarch as component of openSUSE Leap 15.1",
					"product_id": "openSUSE Leap 15.1:python2-markdown2-2.3.7-lp151.2.3.1.noarch"
				},
				"product_reference": "python2-markdown2-2.3.7-lp151.2.3.1.noarch",
				"relates_to_product_reference": "openSUSE Leap 15.1"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python3-markdown2-2.3.7-lp151.2.3.1.noarch as component of openSUSE Leap 15.1",
					"product_id": "openSUSE Leap 15.1:python3-markdown2-2.3.7-lp151.2.3.1.noarch"
				},
				"product_reference": "python3-markdown2-2.3.7-lp151.2.3.1.noarch",
				"relates_to_product_reference": "openSUSE Leap 15.1"
			}
		]
	},
	"vulnerabilities": [
		{
			"cve": "CVE-2020-11888",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2020-11888"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "python-markdown2 through 2.3.8 allows XSS because element names are mishandled unless a \\w+ match succeeds. For example, an attack might use elementname@ or elementname- with an onclick attribute.",
					"title": "Vulnerability description"
				}
			],
			"product_status": {
				"fixed": [
					"openSUSE Leap 15.1:python2-markdown2-2.3.7-lp151.2.3.1.noarch",
					"openSUSE Leap 15.1:python3-markdown2-2.3.7-lp151.2.3.1.noarch"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2020-11888",
					"url": "https://www.suse.com/security/cve/CVE-2020-11888"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1171379 for CVE-2020-11888",
					"url": "https://bugzilla.suse.com/1171379"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Leap 15.1:python2-markdown2-2.3.7-lp151.2.3.1.noarch",
						"openSUSE Leap 15.1:python3-markdown2-2.3.7-lp151.2.3.1.noarch"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 6.1,
						"baseSeverity": "MEDIUM",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:C/C:L/I:L/A:N",
						"version": "3.1"
					},
					"products": [
						"openSUSE Leap 15.1:python2-markdown2-2.3.7-lp151.2.3.1.noarch",
						"openSUSE Leap 15.1:python3-markdown2-2.3.7-lp151.2.3.1.noarch"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2020-05-11T12:19:43Z",
					"details": "moderate"
				}
			],
			"title": "CVE-2020-11888"
		}
	]
}
